ig/_genonce.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
This is part of the HL7 tooling.  If there are changes we'd like to make, we can certainly create a PR against the tooling repo.

Reply:
Just thought I'd ask because of the underscore used in the names.  Another thought I had was to bring these scripts in through the Internet when needed instead of them being baked in our own code base.  This doesn't need to be done in this PR.

Reply:
that's a thought.  it's common for all these scripts to be included in the repo (you can check out some of the HL7 IG repos).  But, that doesn't mean we _have_ to do it that way.  it's a bit of a chicken and egg problem in that the scripts are pulled down as part of the SUSHI init process and then one of the scripts is used to keep the tooling up to date.